scripts/e2e-tests.sh

```diff
@@ -419,6 +433,7 @@ function fail_test() {
   set_test_return_code 1
   [[ -n $1 ]] && echo "ERROR: $1"
   dump_cluster_state
+  dump_metrics
```

Comment:
`dump_cluster_state` is already calling `dump_metrics`, so this line should not be needed?

Reply:
Sorry, where's that?

Reply:
Oh sorry just saw it was called by the `success()` function, the code was collapsed which makes it looks like that it's in the `dump_cluster_state()` function 😅

Reply:
ah, ok cool :) Thanks!!